packages/client/src/classes/client.js

```diff
@@ -72,7 +74,7 @@ class Client {
   createHeaders(data) {
 
     //Merge data with default headers
-    const headers = mergeData(this.defaultHeaders, data);
+    let headers = mergeData(this.defaultHeaders, data);
```

Comment:
Why this change?

Reply:
Ah. Must have forgotten to put it back. Just playing around with the changes.